pandas/core/groupby.py

```diff
@@ -1733,16 +1727,31 @@ class BaseGrouper(object):
     """
     This is an internal Grouper class, which actually holds
     the generated groups
+
+    Parameters
+    ----------
+    axis : the axis to group
```

Comment:
on doc-strings, list the parameter type, then on the next line what it does, e.g

```
axis : int
    the axis to group
...
```